examples/js/effects/VREffect.js

```diff
@@ -67,23 +64,57 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	};
 
-	// fullscreen
+	// fullscreen / VR mode
 
 	var isPresenting = false;
 
 	var canvas = renderer.domElement;
-	var fullscreenchange = canvas.mozRequestFullScreen ? 'mozfullscreenchange' : 'webkitfullscreenchange';
+	var deprecatedAPIMethod;
+	var fullscreenchange;
+	var fullscreenElement;
+
+	if ( canvas.requestFullscreen ) {
+
+		deprecatedAPIMethod = 'requestFullscreen';
+		fullscreenchange = 'fullscreenchange';
+		fullscreenElement = 'fullscreenElement';
+
+	} else if ( canvas.mozRequestFullScreen ) {
+
+		deprecatedAPIMethod = 'mozRequestFullScreen';
+		fullscreenchange = 'mozfullscreenchange';
+		fullscreenElement = 'mozFullScreenElement';
+
+	} else if ( canvas.webkitRequestFullscreen ) {
+
+		deprecatedAPIMethod = 'webkitRequestFullscreen';
+		fullscreenchange = 'webkitfullscreenchange';
+		fullscreenElement = 'webkitFullscreenElement';
+
+	} else if ( canvas.msRequestFullscreen ) {
+
+		fullscreenchange = 'msfullscreenchange';
+		fullscreenElement = 'msFullscreenElement';
 
-	document.addEventListener( fullscreenchange, function () {
+	}
+
+	document.addEventListener( fullscreenchange, function (e) {
 
-		if ( vrHMD && deprecatedAPI ) {
+		if ( fullscreenchange && deprecatedAPI ) {
```

Comment:
I'm confused... Wouldn't `fullscreenchange` be defined if this listener is executed?